libs/@guardian/libs/src/consent-management-platform/sourcepoint.ts

```diff
@@ -195,11 +198,15 @@ export const init = (framework: ConsentFramework, pubData = {}): void => {
 
 	switch (framework) {
 		case 'tcfv2':
-			window._sp_.config.gdpr = {
-				targetingParams: {
-					framework,
-				},
-			};
+			{
+				const pageSection = window.guardian?.config?.page?.section as string;
```

Comment:
Could `isExcludedFromCMP` be updated to accept `string | undefined` so we don't need this typecast? Feels like we shouldn't have to circumvent Typescript here!